crates/networking/p2p/rlpx/connection.rs

```diff
@@ -304,7 +304,11 @@ impl<S: AsyncWrite + AsyncRead + std::marker::Unpin> RLPxConnection<S> {
             tokio::select! {
                 // Expect a message from the remote peer
                 message = self.receive() => {
-                    self.handle_message(message?, sender.clone()).await?;
+                    // TODO: This is a hacky fix for the problem reported in https://github.com/lambdaclass/ethrex/issues/1685
```

Comment:
we should be able to remove this since 1685 has been closed

Reply:
Updated